packages/contracts/test/exchange/core.ts

```diff
@@ -467,35 +445,6 @@ describe('Exchange core', () => {
             expect(orderUtils.getOrderHashHex(signedOrder)).to.be.equal(logArgs.orderHash);
         });
 
-        it('should log 1 event with the correct arguments when order has no feeRecipient', async () => {
-            signedOrder = orderFactory.newSignedOrder({
-                feeRecipientAddress: ZeroEx.NULL_ADDRESS,
-            });
-            const divisor = 2;
-            const res = await exchangeWrapper.fillOrderAsync(signedOrder, takerAddress, {
-                takerAssetFillAmount: signedOrder.takerAssetAmount.div(divisor),
-            });
-            expect(res.logs).to.have.length(1);
-
-            const log = res.logs[0] as LogWithDecodedArgs<FillContractEventArgs>;
-            const logArgs = log.args;
-            const expectedFilledMakerAssetAmount = signedOrder.makerAssetAmount.div(divisor);
-            const expectedFilledTakerAssetAmount = signedOrder.takerAssetAmount.div(divisor);
-            const expectedFeeMPaid = new BigNumber(0);
-            const expectedFeeTPaid = new BigNumber(0);
-
-            expect(signedOrder.makerAddress).to.be.equal(logArgs.makerAddress);
-            expect(takerAddress).to.be.equal(logArgs.takerAddress);
-            expect(signedOrder.feeRecipientAddress).to.be.equal(logArgs.feeRecipientAddress);
-            expect(signedOrder.makerAssetData).to.be.equal(logArgs.makerAssetData);
-            expect(signedOrder.takerAssetData).to.be.equal(logArgs.takerAssetData);
-            expect(expectedFilledMakerAssetAmount).to.be.bignumber.equal(logArgs.makerAssetFilledAmount);
-            expect(expectedFilledTakerAssetAmount).to.be.bignumber.equal(logArgs.takerAssetFilledAmount);
-            expect(expectedFeeMPaid).to.be.bignumber.equal(logArgs.makerFeePaid);
-            expect(expectedFeeTPaid).to.be.bignumber.equal(logArgs.takerFeePaid);
-            expect(orderUtils.getOrderHashHex(signedOrder)).to.be.equal(logArgs.orderHash);
-        });
-
```

Comment:
What do we now expect if there is no feeRecipient?

Reply:
Just a transfer to `address(0)`.